types/wlr_output.c

```diff
@@ -167,6 +170,11 @@ bool wlr_output_set_custom_mode(struct wlr_output *output, int32_t width,
 	if (!output->impl || !output->impl->set_custom_mode) {
 		return false;
 	}
+	if (output->current_mode->width == width &&
```

Comment:
Some outputs may not have modes, in which case `output->current_mode` is NULL. We need to use `output->{width,height,refresh}` instead.

Reply:
Fixed